### PR TITLE
Add vaulted_file test

### PR DESCRIPTION
**Pull request by bcoca**
also fixed some docs for vault_encrypted

##### ISSUE TYPE

- Feature Pull Request

### CODE REVIEW

File: lib/ansible/parsing/vault/__init__.py [hunk 1]
```diff
@@ -103,7 +103,7 @@ def is_encrypted(data):
     return False
 
 
-def is_encrypted_file(file_obj, start_pos=0, count=-1):
+def is_encrypted_file(file_obj, start_pos=0, count=len(b_HEADER)):
```
Comment (mkrizek) [inline lib/ansible/parsing/vault/__init__.py:106]: This change needs to be reflected in the docstring below as well.

File: lib/ansible/plugins/test/core.py [hunk 2]
```diff
@@ -143,6 +144,18 @@ def vault_encrypted(value):
     return getattr(value, '__ENCRYPTED__', False) and value.is_encrypted()
 
 
+def vaulted_file(value):
+    """Evaluate whether a file is a vault
+
+    .. versionadded:: 2.18
+    """
+    try:
+        with open(to_bytes(value), 'rb') as f:
```
Comment (mkrizek) [inline lib/ansible/plugins/test/core.py:153]: Why do we need `to_bytes` on the file name?

File: lib/ansible/plugins/test/vaulted_file.yml [hunk 3]
```diff
@@ -0,0 +1,19 @@
+DOCUMENTATION:
+  name: vaulted_file
+  author: Ansible Core
+  version_added: "2.18"
+  short_description: Is this file an encrypted vault
+  description:
+    - Verifies if the inputed path is an Ansible vault file.
```
Comment (mkrizek) [inline lib/ansible/plugins/test/vaulted_file.yml]:
```suggestion
    - Verifies if the input path is an Ansible vault file.
```

```suggestion
    - Verifies if the inputted path is an Ansible vault file.
```

¯\_(ツ)_/¯

File: lib/ansible/plugins/test/core.py (hunk 2 above)
Comment (bcoca) [reply lib/ansible/plugins/test/core.py:153]: just to be safe when handling 'fun' fs encodings

**Pull request merged**